railties/test/generators/app_generator_test.rb

```diff
@@ -538,18 +525,30 @@ def test_spring
 
   def test_spring_binstubs
     jruby_skip "spring doesn't run on JRuby"
-    generator.stubs(:bundle_command).with('install')
-    generator.expects(:bundle_command).with('exec spring binstub --all').once
-    quietly { generator.invoke_all }
+    command_check = -> command do
+      @install_called ||= 0
```

Comment:
`@install_called` isn't used in the `install` case - did you forget to rename, or shouldn't we expect binstub to be called? :grin: